client/public/icons/hedera.json

```diff
@@ -8,4 +8,4 @@
       "height": 24
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
no need to add the line return manually. If you do want it, add it to the generator script (just append a \n after the JSON.stringify)

Reply:
I didn't really add it but when I was messing with Gradle, this line break came from nowhere so I though I should push it to avoid problems later.. I'll investigate the script to add the line break.